deploy/mainnet/463_deploy_TBTCMetaPoolV3Deposit.ts

```diff
@@ -0,0 +1,63 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+// Deployment names
+const META_POOL_NAME = "SaddleTBTCMetaPoolV3"
+const META_POOL_LP_TOKEN_NAME = `${META_POOL_NAME}LPToken`
+const META_POOL_DEPOSIT_NAME = `${META_POOL_NAME}Deposit`
+const TARGET_META_SWAP_DEPOSIT_NAME = `MetaSwapDeposit`
+const BASE_POOL_NAME = `SaddleBTCPoolV2`
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { execute, get, getOrNull, log, save } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  // Manually check if the pool is already deployed
+  const metaPoolDeposit = await getOrNull(META_POOL_DEPOSIT_NAME)
+  if (metaPoolDeposit) {
+    log(`reusing ${META_POOL_DEPOSIT_NAME} at ${metaPoolDeposit.address}`)
+  } else {
+    const receipt = await execute(
+      "SwapDeployer",
+      {
+        from: deployer,
+        log: true,
+      },
+      "clone",
+      (
+        await get(TARGET_META_SWAP_DEPOSIT_NAME)
+      ).address,
+    )
+
+    const newPoolEvent = receipt?.events?.find(
+      (e: any) => e["event"] == "NewClone",
```

Comment:
+1 on Duy's comments. Should never use loose equality in JS. https://2ality.com/2011/12/strict-equality-exemptions.html